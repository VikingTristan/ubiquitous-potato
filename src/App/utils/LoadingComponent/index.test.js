import React from "react";
import { shallow } from "enzyme";

import LoadingComponent from "./index";

describe("App utils: LoadingComponent", () => {
    it("is defined", () => {
        expect(LoadingComponent).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<LoadingComponent />);

        expect(wrapper).toMatchSnapshot();
    });
});
