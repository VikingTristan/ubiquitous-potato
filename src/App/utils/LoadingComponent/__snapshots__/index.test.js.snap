// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App utils: LoadingComponent renders 1`] = `
<div
  id="loading-component"
>
  <div
    className="loader"
  />
</div>
`;
